setup.py

```diff
@@ -525,7 +525,7 @@ def run(self):
 
     write_version_file()
 
-    with open("README.rst") as f:
+    with open("README.md") as f:
         readme = f.read()
 
     setup(
```

Comment:
To check, but we may also need to add `long_description_content_type="text/markdown",`